javascript/ql/src/experimental/Security/CWE-347/JWTMissingSecretOrPublicKeyVerification.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name JWT Missing Secret Or Public Key Verification
+ * @description The application does not verify the JWT payload with a cryptographic secret or public key.
+ * @kind problem
+ * @problem.severity warning
+ * @precision high
+ * @id js/jwt-missing-verification
+ * @tags security
+ *       external/cwe/cwe-347
+ */
+
+import javascript
+import DataFlow
+
+from CallNode call
+where
+  call = moduleMember("jsonwebtoken", "verify").getACall() and
+  unique(boolean b | b = call.getArgument(1).analyze().getABooleanValue()) = false
+select call.getStartLine(), call,
```

Comment:
```suggestion
import DataFlow
import semmle.javascript.RestrictedLocations

from CallNode call
where
  call = moduleMember("jsonwebtoken", "verify").getACall() and
  unique(boolean b | b = call.getArgument(1).analyze().getABooleanValue()) = false
select call.asExpr().(FirstLineOf),
```

`call.getStartLine()` returns a number, and that doesn't fit with the pattern required by `@kind problem`.  

With my suggestion here, your query should fit the required pattern, and only the first line of the call should be highlighted.  

(You can see the difference by comparing [this query run](https://lgtm.com/query/3011411139123761216/) with [this query run](https://lgtm.com/query/2461261322686219926/)). 
(If you want all lines of the call to be highlighted, then just remove `.asExpr().(FirstLineOf)`).